module/renderer/src/commonTest/kotlin/org/cru/godtools/shared/renderer/content/RenderLinkTest.kt

```diff
@@ -0,0 +1,21 @@
+package org.cru.godtools.shared.renderer.content
+
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.SemanticsNodeInteractionsProvider
+import androidx.compose.ui.test.onNodeWithText
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.ccci.gto.support.androidx.test.junit.runners.AndroidJUnit4
+import org.ccci.gto.support.androidx.test.junit.runners.RunOnAndroidWith
+import org.cru.godtools.shared.tool.parser.model.Link
+import org.cru.godtools.shared.tool.parser.model.Text
+
+@RunOnAndroidWith(AndroidJUnit4::class)
+@OptIn(ExperimentalCoroutinesApi::class, ExperimentalTestApi::class)
+class RenderLinkTest : BaseRenderContentTest() {
```

Comment:
👍

Reply:
if you google compose multiplatform ui testing you should get some documenation on how the node selection (`onNodeWithText()`) works and other things you can test for and ways to select nodes